index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
+
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="index.css">
+    <!-- Template 2-->
+</head>
+
+<body>
+    <header>
+        <div class="header_container">
+            <div class="item flex_one">
+                <h1 class="name">John Doe</h1>
```

Comment:
Kindly use `<a>` tag instead of `<h1>` to make your HTML more semantic. You can find more reasons [here](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) explaining the topic well.